prep-javascript-variables/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    const fullName ="Victoria Dixon";
+    console.log('Full Name',fullName)
+    console.log(typeof fullName);
+    const numberOfPets =2;
+    console.log('Number of Pets', numberOfPets)
+    console.log(typeof numberOfPets)
+    const likesVegetables =true
+    console.log('Likes Vegetables', likesVegetables);
+    console.log(typeof likesVegetables);
+    const nothing=null
+    let never=
```

Comment:
if you are not initializing the variable with a value, you do not need the `=` sign

let never;

would be what you want in this instance.